pywinauto/unittests/test_common_controls.py

```diff
@@ -441,6 +472,55 @@ class ListViewTestCases64(ListViewTestCases32):
         path = os.path.join(mfc_samples_folder, u"RowList.exe")
 
 
+class ListViewWinFormTestCases32(unittest.TestCase):
+
+    """Unit tests for the ListViewWrapper class with WinForm applications"""
+
+    path = os.path.join(winform_folder_32, u"ListView_TestApp.exe")
+
+    def setUp(self):
+        """Set some data and ensure the application is in the state we want"""
+        Timings.Fast()
+
+        app = Application()
+        app.start(self.path)
+
+        self.dlg = app.ListViewEx
+        self.ctrl = app.ListViewEx.ListView.WrapperObject()
+
+    def tearDown(self):
+        """Close the application after tests"""
+        self.dlg.SendMessage(win32defines.WM_CLOSE)
+
+    def testCellInput(self):
+        """Test the ListView GetItem click_input method"""
+        self.ctrl.get_item(0,2).click_input(double=True, where="text")
+        text = "Sometext";
+        self.dlg.TypeKeys(text + "{ENTER}")

```

Comment:
Using an already cached wrapper object ```self.ctrl``` will be a bit faster. Use PEP8 methods names in new tests:
```self.ctrl.type_keys(text + "{ENTER}")```

Same for ```testCellClickInput``` below